### PR TITLE
west.yml: update Zephyr to fd1a129a3e63

### PR DESCRIPTION
**Pull request by LaurentiuM1234**
Contains the following patches affecting SOF:

bba8641354bf dts: xtensa: nxp_imx8: add ESAI0 node
bd9b3c67b240 drivers: dai: add driver for NXP's ESAI
f4c73105e55a drivers: dai: sai: add pinctrl support

Cherry-picked from https://github.com/thesofproject/sof/pull/8863

### CODE REVIEW

**Review by kv2019i — APPROVED** [review]
https://sof-ci.01.org/sofpr/PR9010/build3795/devicetest/index.html is ok (work ongoing to clean up the unavailable DUTs from the CI report)

Let's wait for rest of CI to complete.

**Comment by kv2019i** [comment]
CI is good, just waiting for another review.

**Review by marc-hb — APPROVED** [review]

**Review by iuliana-prodan — APPROVED** [review]

**Pull request merged**
